II.pass-tests/ES-6-practice/es6.js

```diff
@@ -1,29 +1,30 @@
 // Turn this function into an arrow function
-function arrow() {
-  return "es6 is awesome!";
-}
+const arrow = () => {return "es6 is awesome!"};
```

Comment:
Oops, we forgot about this file. We're going to cover ES6 in more depth tomorrow